bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx133-donation-eu-na.html

```diff
@@ -0,0 +1,120 @@
+{#
```

Comment:
**Suggestion:** both donation templates seem almost identical - would it save a lot of repeated code if one extended the other instead?